README.md

```diff
@@ -445,6 +447,70 @@ This repo includes a [GitHub workflow for building a docker image](./.github/wor
 
 The image includes all core WordPress files for the version specified for `WP_VERSION` in the `Dockerfile`, as well as the theme and plugin files necessary for the site. The other element required for the site to run is the database, which is excluded, since each environment should have its own database that is specified by environment variables. This allows local developers to test against local data without interfering with production or staging environments.
 
+#### Accessing the GitHub Container Registry
```

Comment:
Can we add a "Configuration" section similar to Pantheon's [configuration section](https://github.com/sparkbox/sparkpress-wordpress-starter/blob/01418abdaabe5c4481a24c98091fe448d6a05b9f/README.md?plain=1#L472-L486) that lists the secrets/variables that need to be set in order for the workflow to run correctly? We need to list these:

### variables
- `DEPLOY_WITH_DOCKER` - set to `true` to enable workflow

### secrets
- `GH_TOKEN` - a personal access token with write permissions

Reply:
In [validation.yml](https://github.com/sparkbox/sparkpress-wordpress-starter/blob/main/.github/workflows/validation.yml), we use `GITHUB_TOKEN` which is available without needing to set any secrets and it works okay. Is creating a PAT and using `GH_TOKEN` a necessary step because of the write permissions or a nuance with the container registry?

Reply:
Ope, good call. I verified that `GITHUB_TOKEN ` and added a PR for this: https://github.com/sparkbox/sparkpress-wordpress-starter/pull/130